### PR TITLE
get rid of HTML duplication for mobile header

### PR DESCRIPTION
**Pull request by aG0aep6G**
Also replaces the jquery animation for the mobile search box with CSS.

### CODE REVIEW

**Comment by andralex** [comment]
"72 additions and 69 deletions" - not much code removal eh :)

File: css/style.css [hunk 1]
```diff
@@ -1163,7 +1171,25 @@ div#mobile-top, div#mobile-search {
...
+		transition: height 0.5s;
+	}
+
+	div#search-box.open {
+		height: 2.225em;
+		/*
+			= 0.625em [<form>'s margin-top]
+			+ 1.4 em [#search-{query,dropdown,submit}'s height]
+			+ 0.2em [ca. #search-{query,dropdown,submit}'s border]
+		*/
```
Comment (andralex) [inline css/style.css]: is this comment any relevant?

Comment (aG0aep6G) [inline css/style.css]: It explains how the 2.225em above come together.

**Comment by CyberShadow** [comment]
Hmm, such a big patch, but I don't see any difference, visually...

Good!

:+1:

**Pull request merged**